3-networks/README.md

```diff
@@ -9,10 +9,39 @@ The purpose of this step is to:
 
 1. 0-bootstrap executed successfully.
 1. 1-org executed successfully.
-1. 2-environments/envs/[dev|nonprod|prod] executed successfully.
+1. 2-environments executed successfully.
+1. Obtain the value for the access_context_manager_policy_id variable. Can be obtained by running `gcloud access-context-manager policies list --organization YOUR-ORGANIZATION_ID`. It is the value under `name`.
 
 ## Usage
 
-Follow the instructions in the README.md files of each individual 3-networks/envs/ folder.
+### Setup to run via Cloud Build
+
+1. Clone repo `gcloud source repos clone gcp-networks --project=YOUR_CLOUD_BUILD_PROJECT_ID`
+1. Change freshly cloned repo and change to non master branch `git checkout -b plan`
+1. Copy contents of foundation to new repo `cp -R ../terraform-example-foundation/3-networks/* .` (modify accordingly based on your current directory)
+1. Copy cloud build configuration files for terraform `cp ../terraform-example-foundation/build/cloudbuild-tf-* . ` (modify accordingly based on your current directory)
+1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` (modify accordingly based on your current directory)
+1. Ensure wrapper script can be executed `chmod 755 ./tf-wrapper.sh`.
+1. Within each envs/ folder rename terraform.example.tfvars to terraform.tfvars and update the file with values from your environment and bootstrap.
+1. Commit changes with `git add .` and `git commit -m 'Your message'`
+1. Push your plan branch to trigger a plan `git push --set-upstream origin plan`
```

Comment:
nit: this makes it seem like `plan` is a special branch like `prod,dev,nonprod`. We should say any arbitrary branch will trigger plan.
Additionally @rjerrems I was discussing with @daniel-cit offline if it makes sense to have a small separate section where we explain the branching strategy. i.e we care about `prod,dev,nonprod` and these should ideally be protected, how to merge between these etc. WDYT?

Reply:
fixed